app/controllers/services_controller.rb

```diff
@@ -90,4 +112,10 @@ def load_service_and_authorize
       return head(:forbidden)
     end
   end
+
+  def is_admin_of(service)
```

Comment:
@bosulliv42 
May be that function should be implemented in the model instead of the controller
Perhaps we will need it at other places in the application